health-checks.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs');
+const path = require('path');
+const newman = require('newman');
+
+
+const testFolderPath = path.join(__dirname, 'health_check_test_folder');
+
+function runHealthChecks(callback) {
```

Comment:
Could you add a comment explaining what will be tested (access to the file system and the fact that newman correctly reject a call without any file)